bootstrap

```diff
@@ -29,19 +29,19 @@ main(_) ->
     filelib:ensure_dir("_build/bootstrap/lib/rebar/ebin/"),
 
     os:putenv("REBAR_PROFILE", "bootstrap"),
-    compile_vendored(),
```

Comment:
Moved below since `certifi` depends on `parse_trans`, not yet compiled at this moment (unless, that is, we want to also vendor `parse_trans`).